components/chart/parseContentsToNodes.ts

```diff
@@ -36,6 +35,7 @@ function convertContentsToNodes(
       sub_title: contentData.sub_title,
       description: contentData.description,
       city: contentData.city,
+      tags:contentData.tags,
```

Comment:
```suggestion
      tags: contentData.tags,
```
I think you are forgetting formatting the file before saving